Documentation/ViewDesigner/TileConversions.md

```diff
@@ -0,0 +1,106 @@
+# Tile Conversions
+
+## Donut & List
+View Designer has the Donut & List tile as shown below:
+![Donut List](./DonutList.png)
+
+Recreating the tile in workbooks involves two queries, for the Donut portion there are two options.\
+Select **Add query** and paste the original query from View Designer into the cell
+
+**Option 1:** Select Pie Chart from the Visualization Dropdown
+ ![Pie Chart Visualization Menu](./PieChart.png)
+**Option 2:** Add a line to the KQL\
+**Add:** _| render piechart_\
+Note that the Visualization setting should be set to **Set by query**
+ ![Visualization Menu](./SetByQuery.png)
+**Example:**\
+**Original:** search * | summarize AggregatedValue = count() by Type | order by AggregatedValue desc \
+**Updated:** search * | summarize AggregatedValue = count() by Type | order by AggregatedValue desc | render piechart
+
+For creating a list and enabling sparklines please reference the section on [CommonSteps](./CommonSteps.md)
+
+The following is an example of how the Donut & List tile might be reinterpreted in Workbooks
+
+![Donut List Workbooks](./DonutWorkbooks.png)
+
+## Linechart & List
+The original Linechart & List in View Designer looks like the following:
+ 
+![Linechart List](./LineList.png) 
+
+To recreate the Linechart portion we update the query as follows:\
+**Original:** _search * | summarize AggregatedValue = count() by Type_\
+**Updated:** _search * **| make-series Count = count() default=0 on TimeGenerated from {TimeRange:start} to {TimeRange:end} step {TimeRange:grain} by Type**_
+
+There are two options for visualizing the line chart
+
+**Option 1:** Select Line chart from the Visualization dropdown
+ 
+ ![Linechart Menu](./LineViz.png)
+
+**Option 2:** Add a line to the KQL
+Add: | render linechart
+Note that the Visualization setting should be set to Set by query
+
+ ![Visualization Menu](./SetByQuery.png)
+
+**Example:**
+_search * | make-series Count = count() default=0 on TimeGenerated from {TimeRange:start} to {TimeRange:end} step {TimeRange:grain} by Type | render linechart_
+
+For creating a list and enabling sparklines please reference the section on [Common Steps](./CommonSteps.md)
+
+The following is an example of how the Linechart & List tile might be reinterpreted in Workbooks
+
+![Linechart List Workbooks](./LineWorkbooks.png)
+
+## Number & List
+The original View Designer Number & List looks as such:
+ ![Tile List](./TileListEx.png)
+For the number tile, update the query as such:
+
+**Original:** _search * | summarize AggregatedValue = count() by Computer | count_\
+**Updated:** _search *| summarize AggregatedValue = count() by Computer **| summarize Count = count()**_
+
+Then change the Visualization dropdown to Tiles
+ ![Tile Visualization](./TileViz.png)
+Select Tile Settings
+ ![Tile Settings](./TileSet.png)
+From the sidebar menu,\
+Set Visualization: Tiles\
+Tile Settings: \

```

Comment:
The formatting of this came out maybe bad / confusing - check it and see what you think.